File: CourseApi/Models/Course.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CourseApi.Models
{
    public class Course
    {
         public int CourseID { get; set; }

   public string CourseName { get; set; }

   public string Description { get; set; }

   public string Duration { get; set; }

   public int FeesAmount { get; set; }
   public ICollection<Enquiry> Enquiries { get; set; }
    }
}

File: Admin/src/app/model/course.ts
import { Enquiry } from "./enquiry"

export interface Course {
    courseID: number,
    courseName: string,
    description: string,
    duration: string,
    feesAmount: number,
    createdBy: string,
    modifiedBy: string,
    isDelete: boolean
    enquiries?:Enquiry

}
